portal/assignment.py

```diff
@@ -292,3 +281,72 @@ def assignment_grades():
 
         return render_template('layouts/teacher/assignments/assignment-grades.html', assignment=assignment, assignment_name=assignment_name)
     return redirect(url_for('teacher.home'))
+@bp.route('grades/grade-book', methods=('GET', 'POST'))
```

Comment:
For endpoint consistency, use the 'assignments/grade/grade-book' endpoint or move grade-related endpoints to their own file and use 'grade/' as the prefix